src/command_modules/azure-cli-eventgrid/setup.py

```diff
@@ -54,7 +54,10 @@
         'azure',
         'azure.cli',
         'azure.cli.command_modules',
-        'azure.cli.command_modules.eventgrid'
+        'azure.cli.command_modules.eventgrid',
+        'azure.cli.command_modules.eventgrid.sdk',
+        'azure.cli.command_modules.eventgrid.sdk.models',
+        'azure.cli.command_modules.eventgrid.sdk.operations'
```

Comment:
@derekbekoe out of curiosity, why didn't the CI catch this? Doesn't it verify each command package can be installed successfully?